### ISSUE TITLE
`ports.XXX.forwardedHeaders:` and `...proxyProtocol:` ignored if `...tls:` is not true

### ISSUE THREAD

**Issue by alexfouche** [traefik/traefik-helm-chart#867]
### Welcome!

- [X] Yes, I've searched similar issues on [GitHub](https://github.com/traefik/traefik-helm-chart/issues) and didn't find any.
- [X] Yes, I've searched similar issues on the [Traefik community forum](https://community.traefik.io) and didn't find any.

### What version of the Traefik's Helm Chart are you using?

Chart 16.2.0

### What version of Traefik are you using?

does not matter

### What did you do?

`--entrypoints.XXX.forwardedHeaders....=` and `--entrypoints.XXX.proxyProtocol....=` is not added to the arguments of the Traefik command if `ports.XXX.tls:` is not set to `true`

The issue is in `/traefik/templates/_podtemplate.tpl`, because these arguments are enclosed within a `{{- if $config.tls }}` block

1 -> it is impossible to allow `X-Forwarded-For` from an HTTP load balancer for example
2 -> some LB do or can use PROXY protocol even for HTTP (for example Scaleway LB)

### What did you see instead?

`--entrypoints.XXX.forwardedHeaders....=` and `--entrypoints.XXX.proxyProtocol....=` arguments are set for `ports.websecure...` but not for `ports.web...` where i also need them 

### What is your environment & configuration?

n/a

**Comment by alexfouche** [comment]
may be corrected in current version `v.23.1.0` of the chart, but since it implies a lot of changes in Helm values, i could not test it yet

**Comment by mloiseleur** [comment]
Hello @alexfouche ,

I can confirm this issue is fixed with `v23.1.0`. 

With this `values.yaml`:

```yaml
ports:
  web:
    forwardedHeaders:
      trustedIPs: [ "4.4.4.4" ]
  websecure:
    forwardedHeaders:
      trustedIPs: [ "8.8.8.8" ]
```

`Deployment` contains:
```yaml
          - "--entrypoints.web.forwardedHeaders.trustedIPs=4.4.4.4"
          - "--entrypoints.websecure.http.tls=true"
          - "--entrypoints.websecure.forwardedHeaders.trustedIPs=8.8.8.8"
```
